Core/Object Arts/Dolphin/Lagoon/CRTLibraryImportTest.cls

```diff
@@ -0,0 +1,48 @@
+"Filed out from Dolphin Smalltalk 7"!
+
+TestCase subclass: #CRTLibraryImportTest
+	instanceVariableNames: ''
+	classVariableNames: ''
+	poolDictionaries: ''
+	classInstanceVariableNames: ''!
+CRTLibraryImportTest guid: (GUID fromString: '{A3836AC0-E09A-4F44-B33F-3F87F52F2098}')!
+CRTLibraryImportTest comment: ''!
+!CRTLibraryImportTest categoriesForClass!SUnit! !
+!CRTLibraryImportTest methodsFor!
+
+defFilePaths
```

Comment:
It seems a good idea to test this, but the problem is that the VM code is not necessarily available, e.g. for the AppVeyor build. I don't think requiring that one have both repos is really desirable (if that were the case, we may as well merge them into one). Another way to do this will be to check the actual exports of the togo stubs. This is easier than it might at first appear:

> f := FileLocator installRelative localFileSpecFor: 'GuiToGo.exe'.
> l := ExternalLibrary open: f.
> procs := (CRTLibrary selectMethods: [:each | each isKindOf: ExternalMethod]) collect: [:each | each functionName].
> "#getProcAddress: will throw if proc not found, or use getProcAddress:ifAbsent:"
> missing := procs select: [:each | (l getProcAddress: each ifAbsent: []) isNil]
> 
> "-> an IdentitySet('_close' '_finite' 'freopen' '_fpclass')"